build.gradle

```diff
@@ -16,7 +16,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:${versions.androidGradlePlugin}"
+        classpath "com.android.tools.build:gradle:3.2.0"
```

Comment:
❗️This can still use the `versions.androidGradlePlugin` extra property

Reply:
done